lib/swagger.js

```diff
@@ -384,8 +383,6 @@ var SwaggerResource = function(resourceObj, api) {
         }
       }
     };
-    var e = typeof window !== 'undefined' ? window : exports;
-    e.authorizations.apply(obj);
```

Comment:
I believe that if you remove this line, the authorizations won't be applied when fetching an individual api declaration